Extensions/PlatformBehavior/platformerobjectruntimebehavior.js

```diff
@@ -911,6 +920,16 @@ gdjs.PlatformerObjectRuntimeBehavior.prototype.setMaxFallingSpeed = function(
   this._maxFallingSpeed = maxFallingSpeed;
 };
 
+/**
+* Set the maximum speed of Climbing the ladder
+* @param {number} ladderClimbingSpeed The maximum speed of climbing the ladder
+*/
+gdjs.PlatformerObjectRuntimeBehavior.prototype.setladderClimbingSpeed = function (
+  ladderClimbingSpeed
```

Comment:
Can you run Prettier? NOT VS Code autoformatting

Reply:
I switched off both Prettier & VS Code autoformatting :)

Reply:
This is still not good, you can see that other functions are `function(` while yours is `function (`.

This is typical of VS code autformatting. Would be great if you can run Prettier on this file!

Reply:
Alright, I will resolve it right away, I will run prettier and see if it makes the changes that you need.